templates/campaign.tex

```diff
@@ -0,0 +1,100 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Campaign name}{1. Scenario name}{\images/frenzy.pn}
+
+\begin{multicols*}{2}
+
+\subsection*{\MakeUppercase{Scenario length}}
+
+This scenario plays out over \textit{...} rounds.
+
+\subsection*{\MakeUppercase{Player setup}}
+
+\textbf{Faction:} \textit{starting faction ...}
+
+\textbf{Faction Hero:} \textit{starting hero ...}
+
+\textbf{Faction Army:} \textit{starting army ...}
+
+\textbf{Starting Resources:} \textit{starting resources ...}
+
+\textbf{Town Buildings:} \textit{starting town buildings ...}
+
+\textbf{Bonus:} Choose one of the following options: 
+
+\begin{itemize}
+    \item \textit{option 1 ...}
+    \item \textit{option 2 ...}
+\end{itemize}
+
+\subsection*{\MakeUppercase{AI hero setup}}
+
+\textbf{Faction:} \textit{enemy faction ...}
+
+\textbf{Enemy Army:} \textit{enemy army ...}
+
+\textbf{Enemy Deck:} \textit{enemy deck ...}
+
+\textbf{Enemy Spell Deck:} \textit{enemy spell deck ...}
+
+\textbf{Special:} \textit{any special options ...}
+
+\subsection*{\MakeUppercase{Map setup}}
+
+Take the following Map tiles and set them up as shown in the scenario map layout:
+
+\textbf{X × Starting Map tile (I):}
+\begin{itemize}
+    \item \textit{list of tiles ...}
+\end{itemize}
+
+\textbf{Y × Far Map tile (II-III):}
+\begin{itemize}
+    \item \textit{list of tiles ...}
+\end{itemize}
+
+\textbf{Z × Near Map tile (IV-V):}
+\begin{itemize}
+    \item \textit{list of tiles ...}
+\end{itemize}
+
+\subsection*{\MakeUppercase{Heroes placement}}
+
+\textit{Describe how to place your and enemy heroes ...}
+
+\subsection*{\MakeUppercase{Additional rules}}
+
+\textit{Describe additional rules for this campaign ...}
+
+\subsection*{\MakeUppercase{Win/lose conditions}}
+
+\textbf{Win:} \textit{Describe winning conditions ...}
+
+\textbf{Lose:} \textit{Describe losing conditions ...}
+
+\subsection*{\MakeUppercase{Timed events}}
+
+\textbf{1$^{st}$ Round:}
+\begin{itemize}
+    \item Read the "First" section.
```

Comment:
```suggestion
    \item Read the ``First'' section.
```
I know it looks weird, but renders correct quotation marks in accordance to English typography.